src/simulator/simulator.h

```diff
@@ -134,16 +133,23 @@ class Simulator : public ISimulator
 
 
     double startTime() override { return 0.; }
+
     double endTime() override { return finalTime_; }
     double timeStep() override { return dt_; }
     double currentTime() override { return currentTime_; }
 
-    void advance() override
+    void advance(double dt) override
```

Comment:
this is another function directly exposed to python, so could do with a try/catch(print)

Reply:
??

Reply:
similarly to https://github.com/PHAREHUB/PHARE/blob/a5eb3732e486cf5d7b7286e3a9b3f1a21f1122bb/src/simulator/simulator.h#L112

Reply:
ok done